python/ray/experimental/serve/tests/test_serve.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+from pathlib import Path
+import sys
+
+if __name__ == "__main__":
```

Comment:
Yes this is how do you test entire folder!

Reply:
Consider avoiding this in the future, since it removes the advantage of per-test isolation bazel provides. Fine for serve tests since they are so fast atm.